Automattic/sensei
# Remove WooCommerce Components dependency

We were stuck on an old version of `@woocommerce/components` that had some dependencies that would prevent us from updating our build system. The new version of `@woocommerce/components` gets pretty complicated with the new standard color schemes and also had some dependencies that were causing issues. 

### Changes proposed in this Pull Request

* Brings over the following components from `@woocommerce/components`: `List`, `Stepper`, `Section`/`H`. 
  * I'm not sure if we're always using `Section` and `H` as intended, but I didn't get into it too much. If we know the depth and there isn't a `<Section>`, we should probably just use the explicit HTML header tag.
* Replaces `<Card>` and `<Spinner>` with components in `@wordpress/components`. Note: the "elevated" (shadow) style  on the `<Card>` component is slightly darker. I thought it looked fine, but if folks preferred the WooCommerce variant I can bring it over.
* Replaces `<Link>` with ...`<a>`. It doesn't seem like that component did anything for non-WC admin links. If I didn't see something, let me know.
* We can eventually probably replace the stepper in the data port functionality with this one, but for now I didn't touch it.

### Testing instructions

* Go through the setup wizard and make sure the slight changes are acceptable.
* Go through import/export wizards and make sure there aren't any regressions.
